crates/solver/src/settlement_submission/submitter.rs

```diff
@@ -481,22 +481,19 @@ impl<'a> Submitter<'a> {
                     || gas_price.max_fee_per_gas < replacement_price.max_fee_per_gas
                 {
                     tracing::debug!(
-                        ?gas_price,
-                        ?replacement_price,
                         sleep = ?params.retry_interval,
-                        "keep waiting for gas price to increase enough"
+                        "keep waiting for gas price to increase enough, gas price={}, replacement price={}",
+                        gas_price,
+                        replacement_price,
```

Comment:
nit, I would still use `tracing` module here. You can use `%` instead of `?` to format with `Display` instead of `Debug`.

```rust
                    tracing::debug!(
                        %gas_price,
                        %replacement_price,
                        sleep = ?params.retry_interval,
                        "keep waiting for gas price to increase enough"
                        "keep waiting for gas price to increase enough",
```

Reply:
Fixed.